libs/sdk-ui-kit/src/Header/Header.tsx

```diff
@@ -229,7 +229,7 @@ class AppHeaderCore extends Component<IAppHeaderProps & WrappedComponentProps, I
                 />
             );
         }
-        if (trialDaysLeft > 30 && trialDaysLeft < 90) {
+        if (trialDaysLeft > 30 && trialDaysLeft - 1 <= 90) {
```

Comment:
Why don't we simply write as below?

```
if (trialDaysLeft > 30 && trialDaysLeft - 1 <= 90) {
```

↓

```
if (trialDaysLeft > 30 && trialDaysLeft <= 91) { // expiredDate is one day after the last day that user can use the service
```

Reply:
Fixed. Please check again. Thank you.